src/main/java/com/checkout/payments/request/source/apm/RequestCvConnectSource.java

```diff
@@ -0,0 +1,27 @@
+package com.checkout.payments.request.source.apm;
+
+import com.checkout.common.Address;
+import com.checkout.common.PaymentSourceType;
+import com.checkout.payments.request.source.AbstractRequestSource;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@EqualsAndHashCode(callSuper = true)
```

Comment:
We should add the getter and setter annotations here

Reply:
That's right, I forgot how to program on java :(